pynecone/components/forms/password.py

```diff
@@ -0,0 +1,12 @@
+"""An Password component."""
+
+from typing import Set
+from .input import Input
+from pynecone.components.component import EVENT_ARG
+from pynecone.var import Var
+
+
+class Password(Input):
+    """The Password component is a component that is used to get password input from users."""
+
+    type_: Var[str] = "password"
```

Comment:
I think you need a `#type: ignore` at the end of this line to make Pyright happy

Reply:
Okay will do that.

Reply:
I fixed it and pyright is now happy :D